### DISCUSSION TITLE
How do I uninstall Geode without trouble?

### DISCUSSION THREAD

**Discussion by LowoTV** [geode-sdk/sdk#126] [Help] [answered]
I was trying to uninstall **Geode** by moving _the files to a folder_ I use for _**unused things**_ so then _I moved the files named with the word "Geode"_ then _it gave me an error_ that **It will very likely crash** and **_it crashed 2 of 2 times I ran it_** and I couldn't find a way to uninstall it.

### So please, _how do I uninstall it?_

**I appreciate the mod** but beforehand _I already had a mod loader (GDHM)_ and had around **10 mods** on it then I saw Geode and installed it.

One more thing. In future versions of Geode, **add an uninstall program or uninstall button in the game** so that **it will be easier to uninstall late when not needed**

**Comment by matcool** [comment, marked as answer]
you can easily remove geode from a gd instance by deleting geode.dll and xinput9_1_0.dll from the gd folder

if you want to remove all the geode leftovers, also delete the geode folder and GeodeBootstrapper.dll, as well as the geode folder in %localappdata%/GeometryDash

**Reply by LowoTV** [reply]
what about **geode.lib**? is it _deletable_ as well?

**Reply by matcool** [reply]
its completely unused, dont even know why installer puts it there

**Reply by LowoTV** [reply]
ig its **deletable** _idk_

**Comment by matcool** [comment]
the installer also supports uninstalling, so you can just run it again and give it the same gd path

**Reply by LowoTV** [reply]
ok
